staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -432,6 +432,15 @@ type ListOptions struct {
 	SendInitialEvents *bool `json:"sendInitialEvents,omitempty" protobuf:"varint,11,opt,name=sendInitialEvents"`
 }
 
+const (
+	// InitialEventsAnnotationKey the name of the key
+	// under which an annotation marking the end of
+	// a watchlist stream is stored.
+	//
+	// The annotation is added to a "Bookmark" event.
+	InitialEventsAnnotationKey = "k8s.io/initial-events-end"
```

Comment:
for annotations we use kubernetes.io. I found the introduction in https://github.com/kubernetes/kubernetes/pull/120872/files which accidentally bypassed review that should have caught this.  Since this is already baked into clients, we're stuck.